tests/Integration/Framework/MessageBuilder.php

```diff
@@ -72,7 +74,7 @@ public function to($to) {
 	 * @param string $cc
 	 * @return MessageBuilder
 	 */
-	public function cc($cc) {
+	public function cc(string $cc) {
```

Comment:
```suggestion
	public function cc(?string $cc) {
```

Reply:
This doesn't make much sense IMO. If there is no CC, this method shouldn't be called

Reply:
Maxence